.github/workflows/deploy-website-next.yml

```diff
@@ -0,0 +1,77 @@
+name: Finish deploying to playground.wordpress.net (don't use manually)
+# Sorry, the previous job just built it and uploaded it to GitHub
+
+on:
+    workflow_run:
+        workflows: [Deploy to playground.wordpress.net]
+        types:
+            - completed
+
+jobs:
+    deploy_to_wp_cloud:
+        # Only run this workflow from the trunk branch and when it's triggered by another workflow OR certain maintainers
+        if: >
+            github.ref == 'refs/heads/trunk' && (
+                github.event.workflow_run.conclusion == 'success' ||
+                github.actor == 'adamziel' ||
+                github.actor == 'dmsnell' ||
+                github.actor == 'bgrgicak' ||
+                github.actor == 'brandonpayton'
+            )
+
+        # Specify runner + deployment step
+        runs-on: ubuntu-latest
+        environment:
+            name: playground-wordpress-net-wp-cloud
+        steps:
+            - name: Wait for latest build artifact
+              shell: bash
+              # Sleep to give the GitHub API time to register the artifact,
+              # otherwise the artifact will not be available when the webhook is called
+              run: |
+                  while true; do
+                      API_HASH=$(curl 'https://api.github.com/repos/wordpress/wordpress-playground/actions/artifacts?name=playground-website&per_page=2' \
+                          | jq -r '.artifacts[0].workflow_run.head_sha')
+                      if [ "$API_HASH" = "$GITHUB_SHA" ]; then
+                          break;
+                      fi;
+                      echo "$API_HASH was not $GITHUB_SHA, waiting 10 seconds...";
+                      sleep 10;
+                  done;
```

Comment:
if this step using another CI job? can we [reuse the workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows) and avoid this waiting code?

Reply:
@dmsnell, this is indeed [duplicated code from deploy-website.yml](https://github.com/WordPress/wordpress-playground/blob/3b74b4c31534b59f122ef36641aa7daeaca2f905/.github/workflows/deploy-website.yml#L35-L43), but once we are done migrating to the new host, I imagine we will remove the original workflow and leave this as the only remaining "waiting block".

I'm not sure if there is a way to avoid the waiting entirely... Are you thinking the waiting code may not be required at all if we take another approach, or do you just mean that we could avoid duplicating it?

Reply:
I meant that maybe Github will schedule this Job once the previous one finishes, and we won't have to sit in a 10s wait loop. I have no idea if that's possible

Reply:
Ah, thanks. That's actually what this bit does:
https://github.com/WordPress/wordpress-playground/pull/1293/files#diff-e4927ad07277ff5004af1672f9a870dcf7c859ac71348aefd357ef9f30348d51R4-R8

A confusing thing is that the name within "build-website.yml" its name is "Deploy to playground.wordpress.net", which is the name the actual deploy workflow triggers on:
```yml
on:
     workflow_run:
         workflows: [Deploy to playground.wordpress.net]
         types:
             - completed
```

Apparently, based on the current deploy workflow, GH API still takes some time to reflect an artifact uploaded by the previous workflow, but I have not tested that assumption.

Reply:
The only reason building a website and deploying a website are separate workflows are GitHub limitations around downloading artifacts. Not going too deep into details, in this kind of "push" scenario where we build the website and are able to immediately rsync it, we can have a single workflow and don't use artifacts at all.

Reply:
👍 Thanks for explaining. It sounds like we can initially ship this separate workflow and merge it with the build workflow once playground.wordpress.net is totally moved to the new host.

Though it is not required, do you see any value in just saving a build artifact to keep a record of what was actually deployed?

Reply:
> Though it is not required, do you see any value in just saving a build artifact to keep a record of what was actually deployed?

@brandonpayton Not at the moment:

* I never had to go back to review previous artifacts
* The current deployment workflow keeps a copy of the last website version for quick rollbacks if needed
* Rebuilding a historical commit is fast

I'm trying to think of a scenario where this would be useful and nothing comes up. Am I missing anything?

Reply:
@adamziel, I don't think you're missing anything. I asked because I like reproducibility but didn't really have an opinion on whether it mattered here. :) Thanks, we can just plan to drop the artifact in the future.